docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,885 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    naive_DAMP(T, m, split_index)  # dummy run
</pre><p>I added a dummy run to make sure the running time does not contain the compilation time of njit funcs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>